include/zephyr/bluetooth/bluetooth.h

```diff
@@ -577,11 +571,13 @@ struct bt_data {
  * @brief Get the total size (in bytes) of a given set of @ref bt_data
  * structures.
  *
+ * The total size includes the length (1 byte) and type (1 byte) fields for each element, plus their
+ * respective data lengths.
```

Comment:
Possible future change: We probably should use `octet` instead of `byte`

Reply:
Changed the occurences with the exception of in `BT_DATA_BYTES`